src/app/markers.ts

```diff
@@ -59,9 +60,14 @@ export function roundMarkerFromRoundFlower(
   barcodeValue: number,
   currentBeePollen: number,
   recentFlowerInteractions: Interaction[],
-  incompatibleFlower: boolean
+  bee: BeeSpecies
 ): RoundMarker {
-  const lastVisitedIncompatible = recentFlowerInteractions[0]?.incompatibleFlower && recentFlowerInteractions[0]?.barcodeValue === barcodeValue
+  const incompatibleFlower = !bee.flowers_accepted.map(acceptedFlower =>
+    acceptedFlower.id).includes(flower.species.id);
```

Comment:
This is not the best place to split this line. It might be just fine to have this on a single line too.

Reply:
Doesn't break the TSLint length, but It looks a little long on a single line.